lib/usd/translators/meshReader.cpp

```diff
@@ -127,12 +127,13 @@ bool MayaUsdPrimReaderMesh::Read(UsdMayaPrimReaderContext* context)
         }
     }
 
-    // assign material
-    assignMaterial(mesh, _GetArgs(), meshRead.meshObject(), context);
-
     // assign primvars to mesh
     UsdMayaMeshReadUtils::assignPrimvarsToMesh(
         mesh, meshRead.meshObject(), _GetArgs().GetExcludePrimvarNames());
+
```

Comment:
UV needs to be read and present in the mesh before the material is bound to insure proper UV linkage.